src/Helpers/AuditResolver.php

```diff
@@ -7,11 +7,7 @@
 
 class AuditResolver
 {
-    /**
-     * @param Model|null $model
-     * @return int|null
-     */
-    public static function findUserId($model): ?int
+    public static function findUserId(Model|null $model): ?int
```

Comment:
What about nullable type hint?
```suggestion
    public static function findUserId(?Model $model): ?int
```